run.go

```diff
@@ -80,8 +82,24 @@ func TestingT(testingT *testing.T) {
 // provided run configuration.
 func RunAll(runConf *RunConf) *Result {
 	result := Result{}
+	skipTests := false
 	for _, suite := range allSuites {
-		result.Add(Run(suite, runConf))
+		var res *Result
+		// a test from one suite has failed and we loop through
```

Comment:
Comment is misplaced (nothing has failed here). Should be below, perhaps shorter: